### PR TITLE
Proposed working group changes

### PR DESCRIPTION
**Pull request by jmertic**
@kdt3rd Please review

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

### CODE REVIEW

File: process/working_groups.md [hunk 1]
```diff
@@ -6,24 +6,27 @@ parent: Processes
...
 _Modeled off of CNCF Working Group model ( https://github.com/cncf/toc/blob/master/workinggroups/README.md )_
 
-## Introduction
-
 The purpose of working groups are to study and report on a particular question and make recommendations based on it's findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing it's findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a [TAC member] and with an affirmative vote of the ASWF TAC. The TAC can also shut down a working group with a super majority vote.
 
-## Process
+## Forming a working group
+
+Interested parties in forming a working groups should align with the interested parties on the [charter template]. Once the working group charter is agreed upon, and a [TAC member] is identified as a sponsor, it should be shared with the [TAC email list] to be added to the agenda for the next TAC meeting for review and approval. A working group will not be designated and recognized as an official ASWF Working Group until the TAC approves the charter.
```
Comment (jfpanisset) [inline process/working_groups.md:13]: I'm not sure I understand that first sentence. Are there two sets of "interested parties"? Also "forming a working groups" should either be "forming a working group" or "forming working groups", but probably the former. How about:

Parties interested in forming a working group should use the [charter template] to define a charter to be proposed to the TAC.

Does that capture it?

File: process/working_groups.md [hunk 2]
```diff
@@ -6,24 +6,27 @@ parent: Processes
...
+
+## Working Group activities and collaboration
+
+Working groups generally will meet on a regular basis and operate in an open and transparent way. ASWF provides working groups access to Zoom, a mailing list, Slack channel, wiki space, and GitHub repo. Please submit a [request on the foundation repo](https://github.com/AcademySoftwareFoundation/foundation/issues/new) to have this infrastructure created.
```
Comment (jfpanisset) [inline process/working_groups.md:17]: Is that the best way to go? No issues have been filed so far against that repo, and I'm afraid few people know it exists. Would a LF ticket be a more direct way?

File: process/working_groups.md [hunk 3]
```diff
@@ -6,24 +6,27 @@ parent: Processes
...
+
+Working groups should operate within the scope of the mission, goals, non-goals, and deliverables listed in the approved charter. Any changes in scope must be reflected in the working group charter and approved by the TAC.
 
-If you would like to submit a working group proposal, please submit the [template] to the TAC. Working groups can begin formation to complete the [template] before TAC approval, but will not have official designation as an ASWF Working Group until TAC approval. You will also have to present to the ASWF TAC and wider community before your WG proposal will be voted upon by the TAC. 
+## Review
 
-## FAQs
+Working Groups will present to the TAC it's progress towards the mission, goals, non-goals, and deliverables listed in the approved charter after 6 months from when the working group was formally approved. This is an opportunity for the working group to elevate any concerns or needs from the TAC to complete it's mission, goals, non-goals, and deliverables. If there is insufficent progress within the working group, the TAC may opt to close the working group with a 2/3 majority vote of [TAC member]s.
```
Comment (jfpanisset) [inline process/working_groups.md:23]: it's -> its (2x)

Comment (jfpanisset) [reply process/working_groups.md:23]: Progress on non-goals seems like an odd requirement?

File: process/working_groups.md [hunk 4]
```diff
@@ -6,24 +6,27 @@ parent: Processes
...
 
-### Q: Who can propose a new working group?
-A: Anyone can propose a new working group ( whether from an ASWF member organization or not ), but it must be sponsored by a [TAC member].
+After one year, the working group is expected to have completed the mission, goals, non-goals, and deliverables listed in the approved charter. If the working group has not completed it's mission, goals, non-goals, and deliverables, and it's of the opinion of the TAC that it can be completed, the TAC may opt to extend the working group for up to 1 year with a 2/3 majority vote of [TAC member]s.
```
Comment (jfpanisset) [inline process/working_groups.md:25]: it's -> its (2x)

File: process/working_groups.md [hunk 5]
```diff
@@ -6,24 +6,27 @@ parent: Processes
...
 
-### Q: Can working groups have infrastructure during formation to enable collaboration?
-A: Yes! Please submit a [request on the foundation repo](https://github.com/AcademySoftwareFoundation/foundation/issues/new) and we can provide a mailing list, Slack channel, and GitHub repo.
+## Long-term working group
 
-### Q: What if the goals or mission of the working group change?
-A: The TAC recognizes that in the lifecycle of a working group goals or mission may change. In those cases, the working group must present to the TAC an updated version of their [template] for approval.
+The TAC may also opt to establish the working group as a long-term working group after it's 1 year anniversary with a 2/3 majority vote of [TAC member]s, if it is deemed that such a working group is essential for the sustainability of the Academy Software Foundation. It is expected that a long-term working group has it's charter reviewed and revisted annually with the approval of the TAC.
```
Comment (jfpanisset) [inline process/working_groups.md:29]: it's -> its (2x)

This addresses the concerns about "long term working groups" well.

**Review by jfpanisset — COMMENTED** [review]
I think this is in good shape and not controversial, with the small issues I brought up below addressed, perhaps this should be taken out of draft stage and potentially merged?

File: process/working_groups.md (hunk 1 above)
Comment (jmertic) [reply process/working_groups.md]: Fixed now - thanks!

File: process/working_groups.md (hunk 2 above)
Comment (jmertic) [reply process/working_groups.md:17]: I'm trying to drive more folks this way as it's easier to track and manage. The LF ticket ends up being more useful for those cases where other LF departments need to engage.

File: process/working_groups.md (hunk 5 above)
Comment (jmertic) [reply process/working_groups.md]: Fixed typo - thanks for the feedback on that statement!

**Review by jfpanisset — APPROVED** [review]
Looks good! Is it time to remove the "draft" status and allow this to go in? Maybe prompt on the #tac Slack channel for one last set of reviews?

**Comment by jmertic** [comment]
@kdt3rd Indicated the TAC was generally good with these changes - so merging them in now.

**Pull request merged**